### ISSUE TITLE
clerk/build! doesn't like defrecord?

### ISSUE THREAD

**Issue by T045T** [nextjournal/clerk#329]
If I put this into `test.clj`

```clj
^{:nextjournal.clerk/toc true}
(ns defrecord-repro
  "clerk/build! doesn't like defrecord :("
  (:require [nextjournal.clerk :as clerk]))

(defrecord Human
    [arms
     legs
     heads])

;; Normal human being
(def brad (->Human 2 2 3))
```

I get errors trying to build a static version of my notebook (the below is in my jacked-in CIDER REPL):

```
user> (clerk/build! {:paths ["test.clj"] :out-path "test"})
Clerk evaluated 'jar:file:/Users/nilsberg/.m2/repository/io/github/nextjournal/clerk/0.12.699/clerk-0.12.699.jar!/nextjournal/clerk/builder_ui.clj' in 198.802268ms.
👷🏼 Clerk is building 1 notebooks…
🧐 Parsing… Done in 1.999ms. ✅
🔬 Analyzing… Done in 83.316ms. ✅
🔨 Building "test.clj"… Execution error (AbstractMethodError) at defrecord_repro.Human/containsKey (REPL:-1).
Method defrecord_repro/Human.containsKey(Ljava/lang/Object;)Z is abstract
```

Full stack trace:

```

  Show: Project-Only All 
  Hide: Clojure Java REPL Tooling Duplicates  (16 frames hidden)

1. Unhandled java.lang.AbstractMethodError
   Method defrecord_repro/Human.containsKey(Ljava/lang/Object;)Z is
   abstract

                      REPL:   -1  defrecord-repro.Human/containsKey
                   RT.java:  834  clojure.lang.RT/contains
                  core.clj: 1506  clojure.core/contains?
                  core.clj: 1498  clojure.core/contains?
               viewer.cljc:  108  nextjournal.clerk.viewer$wrapped_value_QMARK_/invokeStatic
               viewer.cljc:  104  nextjournal.clerk.viewer$wrapped_value_QMARK_/invoke
               viewer.cljc: 1055  nextjournal.clerk.viewer$apply_viewers_STAR_/invokeStatic
               viewer.cljc: 1046  nextjournal.clerk.viewer$apply_viewers_STAR_/invoke
               viewer.cljc: 1221  nextjournal.clerk.viewer$present_STAR_/invokeStatic
               viewer.cljc: 1216  nextjournal.clerk.viewer$present_STAR_/invoke
               viewer.cljc: 1323  nextjournal.clerk.viewer$present/invokeStatic
               viewer.cljc: 1319  nextjournal.clerk.viewer$present/invoke
               viewer.cljc: 1321  nextjournal.clerk.viewer$present/invokeStatic
               viewer.cljc: 1319  nextjournal.clerk.viewer$present/invoke
               viewer.cljc:  437  nextjournal.clerk.viewer$transform_result/invokeStatic
               viewer.cljc:  429  nextjournal.clerk.viewer$transform_result/invoke
                  core.clj: 6231  clojure.core/update
                  core.clj: 6223  clojure.core/update
                  AFn.java:  160  clojure.lang.AFn/applyToHelper
               RestFn.java:  132  clojure.lang.RestFn/applyTo
                  core.clj:  673  clojure.core/apply
                  core.clj:  662  clojure.core/apply
               viewer.cljc:  342  nextjournal.clerk.viewer$update_val$fn__12481/invoke
                  core.clj: 2586  clojure.core/comp/fn
               viewer.cljc: 1052  nextjournal.clerk.viewer$apply_viewers_STAR_/invokeStatic
               viewer.cljc: 1046  nextjournal.clerk.viewer$apply_viewers_STAR_/invoke
                  core.clj: 2586  clojure.core/comp/fn
                  core.clj: 2586  clojure.core/comp/fn
                  core.clj: 2759  clojure.core/map/fn/fn
                  core.clj: 7682  clojure.core/preserving-reduced/fn
     PersistentVector.java:  343  clojure.lang.PersistentVector/reduce
                  core.clj: 6885  clojure.core/reduce
                  core.clj: 7693  clojure.core/cat/fn
                  core.clj: 2759  clojure.core/map/fn/fn
     PersistentVector.java:  343  clojure.lang.PersistentVector/reduce
                  core.clj: 6946  clojure.core/transduce
                  core.clj: 6962  clojure.core/into
                  core.clj: 6950  clojure.core/into
                  core.clj: 2648  clojure.core/partial/fn
                  core.clj: 6231  clojure.core/update
                  core.clj: 6223  clojure.core/update
               viewer.cljc:  894  nextjournal.clerk.viewer$process_blocks/invokeStatic
               viewer.cljc:  891  nextjournal.clerk.viewer$process_blocks/invoke
                  core.clj: 2641  clojure.core/partial/fn
                  core.clj: 6231  clojure.core/update
                  core.clj: 6223  clojure.core/update
               viewer.cljc:  906  nextjournal.clerk.viewer$fn__12685/invokeStatic
               viewer.cljc:  904  nextjournal.clerk.viewer$fn__12685/invoke
               viewer.cljc: 1052  nextjournal.clerk.viewer$apply_viewers_STAR_/invokeStatic
               viewer.cljc: 1046  nextjournal.clerk.viewer$apply_viewers_STAR_/invoke
               viewer.cljc: 1221  nextjournal.clerk.viewer$present_STAR_/invokeStatic
               viewer.cljc: 1216  nextjournal.clerk.viewer$present_STAR_/invoke
               viewer.cljc: 1323  nextjournal.clerk.viewer$present/invokeStatic
               viewer.cljc: 1319  nextjournal.clerk.viewer$present/invoke
               viewer.cljc: 1321  nextjournal.clerk.viewer$present/invokeStatic
               viewer.cljc: 1319  nextjournal.clerk.viewer$present/invoke
                  view.clj:   13  nextjournal.clerk.view/doc->viewer
                  view.clj:    9  nextjournal.clerk.view/doc->viewer
               builder.clj:  306  nextjournal.clerk.builder/build-static-app!/fn/fn
               builder.clj:  303  nextjournal.clerk.builder/build-static-app!/fn
                  core.clj: 2777  clojure.core/map/fn
              LazySeq.java:   42  clojure.lang.LazySeq/sval
              LazySeq.java:   51  clojure.lang.LazySeq/seq
                   RT.java:  535  clojure.lang.RT/seq
                  core.clj:  139  clojure.core/seq
             protocols.clj:   24  clojure.core.protocols/seq-reduce
             protocols.clj:   75  clojure.core.protocols/fn
             protocols.clj:   75  clojure.core.protocols/fn
             protocols.clj:   13  clojure.core.protocols/fn/G
                  core.clj: 6886  clojure.core/reduce
                  core.clj: 6958  clojure.core/into
                  core.clj: 6970  clojure.core/mapv
                  core.clj: 6970  clojure.core/mapv
               builder.clj:  300  nextjournal.clerk.builder/build-static-app!
               builder.clj:  273  nextjournal.clerk.builder/build-static-app!
                 clerk.clj:  459  nextjournal.clerk/build!
                 clerk.clj:  417  nextjournal.clerk/build!
                      REPL:  401  user/eval51740
                      REPL:  401  user/eval51740
             Compiler.java: 7194  clojure.lang.Compiler/eval
             Compiler.java: 7149  clojure.lang.Compiler/eval
                  core.clj: 3215  clojure.core/eval
                  core.clj: 3211  clojure.core/eval
    interruptible_eval.clj:   87  nrepl.middleware.interruptible-eval/evaluate/fn/fn
                  AFn.java:  152  clojure.lang.AFn/applyToHelper
                  AFn.java:  144  clojure.lang.AFn/applyTo
                  core.clj:  667  clojure.core/apply
                  core.clj: 1990  clojure.core/with-bindings*
                  core.clj: 1990  clojure.core/with-bindings*
               RestFn.java:  425  clojure.lang.RestFn/invoke
    interruptible_eval.clj:   87  nrepl.middleware.interruptible-eval/evaluate/fn
                  main.clj:  437  clojure.main/repl/read-eval-print/fn
                  main.clj:  437  clojure.main/repl/read-eval-print
                  main.clj:  458  clojure.main/repl/fn
                  main.clj:  458  clojure.main/repl
                  main.clj:  368  clojure.main/repl
               RestFn.java: 1523  clojure.lang.RestFn/invoke
    interruptible_eval.clj:   84  nrepl.middleware.interruptible-eval/evaluate
    interruptible_eval.clj:   56  nrepl.middleware.interruptible-eval/evaluate
    interruptible_eval.clj:  152  nrepl.middleware.interruptible-eval/interruptible-eval/fn/fn
                  AFn.java:   22  clojure.lang.AFn/run
               session.clj:  218  nrepl.middleware.session/session-exec/main-loop/fn
               session.clj:  217  nrepl.middleware.session/session-exec/main-loop
                  AFn.java:   22  clojure.lang.AFn/run
               Thread.java:  748  java.lang.Thread/run
```

**Comment by T045T** [comment]
An interesting thing about this is that `clerk/show!` works on this file, up until I try to run `clerk/build!`, so whatever is breaking is getting cached somewhere.

**Comment by mk** [comment]
I believe this has also been fixed with e5c00bb0473bfdf1562024879a5ff69ed3309477. Please reopen if it's still an issue for you.
